bridge/irc_manager.go

```diff
@@ -153,7 +155,7 @@ func (m *IRCManager) HandleUser(user DiscordUser) {
 
 	nick := m.generateNickname(user)
 
-	innerCon := irc.IRC(nick, "discord")
+	innerCon := irc.IRC(nick, user.Username)
```

Comment:
Please can you make this a setting? It could be an optional setting `puppet_username` which defaults to the empty string. and if the `config.PuppetUsername` is the empty string, instead it would use a **sanitised version** of `user.Username`

(I think this needs sanitising? Correct me if I'm wrong.)

Reply:
Im unsure if sanitisation would be required but better safe then sorry. will update

Reply:
Updated to sanitise usernames. i reused the `sanitiseNickname` function as that seems more sensible then copying it